modules/heat_conduction/src/materials/ElectricalConductivity.C

```diff
@@ -16,18 +16,20 @@ InputParameters
 ElectricalConductivity::validParams()
 {
   InputParameters params = Material::validParams();
-  params.addCoupledVar("temp", 300.0, "variable for temperature");
+  params.addCoupledVar("temp", 293.0, "variable for temperature");
```

Comment:
Could I talk you into the parameter names `temperature` here and `reference_temperature` on line 30 please?

Reply:
Done - also expanded upon the names of the other parameters for clarity.